### PR TITLE
Settings: Add 'Site Tools' header to delete-site-options

### PR DESCRIPTION
**Pull request by seear**
First step of #12169.

Adds a header and updates the text and styling of the various existing settings at the bottom of _General_ (currently known as delete-site-options).

**Before**
screen shot 2017-05-19 at 12 48 30

**After**
screen shot 2017-05-22 at 16 40 54

**Mobile**
screen shot 2017-05-22 at 16 41 38

**Next steps for #12169** (in separate PRs)
* Rename `<DeleteSiteOptions/>` component to `<SiteOptions>`
* Bring _Import_ & _Export_ settings into the component

**To Test**
* The settings at http://calypso.localhost:3000/settings/general/{site} should work exactly as before

### CODE REVIEW

File: client/my-sites/site-settings/delete-site-options/index.jsx [hunk 1]
```diff
@@ -11,68 +11,63 @@ import CompactCard from 'components/card/compact';
...
+			deleteSite: translate( 'Delete Site' )
 		};
 
 		if ( ! this.props.hasLoadedSitePurchasesFromServer ) {
 			return null;
 		}
 
 		if ( ! config.isEnabled( 'upgrades/domain-search' ) ) {
-			changeAddressLinkText = this.translate( 'Change your site\'s address.' );
+			changeAddressLinkText = translate( 'Change your site address.' );
```
Comment (a8ci18n) [inline client/my-sites/site-settings/delete-site-options/index.jsx]: Hi! I've found a possible matching string that has already been translated **38** times:
`translate( 'Change Site Address' )` *ES Score: 9*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/settings-site-tools)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

File: client/my-sites/site-settings/delete-site-options/index.jsx [hunk 2]
```diff
@@ -11,68 +11,63 @@ import CompactCard from 'components/card/compact';
...
-		const themeSetupLinkText = this.translate( 'Make your site look like your theme\'s demo.' );
+		let changeAddressLinkText = translate( 'Register a new domain or change your site\'s address.' );
+		const themeSetupLinkText = translate( 'Make your site look like your theme\'s demo.' );
 		const strings = {
-			changeSiteAddress: this.translate( 'Change Site Address' ),
-			themeSetup: this.translate( 'Theme Setup' ),
-			startOver: this.translate( 'Start Over' ),
-			deleteSite: this.translate( 'Delete Site' )
+			changeSiteAddress: translate( 'Change your site address' ),
+			themeSetup: translate( 'Theme setup' ),
```
Comment (a8ci18n) [inline client/my-sites/site-settings/delete-site-options/index.jsx]: :disappointed: 17 existing translations will be lost with this change.

File: client/my-sites/site-settings/delete-site-options/index.jsx [hunk 3]
```diff
@@ -11,68 +11,63 @@ import CompactCard from 'components/card/compact';
...
-		let changeAddressLinkText = this.translate( 'Register a new domain or change your site\'s address.' );
-		const themeSetupLinkText = this.translate( 'Make your site look like your theme\'s demo.' );
+		let changeAddressLinkText = translate( 'Register a new domain or change your site\'s address.' );
+		const themeSetupLinkText = translate( 'Make your site look like your theme\'s demo.' );
 		const strings = {
-			changeSiteAddress: this.translate( 'Change Site Address' ),
-			themeSetup: this.translate( 'Theme Setup' ),
-			startOver: this.translate( 'Start Over' ),
-			deleteSite: this.translate( 'Delete Site' )
+			changeSiteAddress: translate( 'Change your site address' ),
```
Comment (a8ci18n) [inline client/my-sites/site-settings/delete-site-options/index.jsx]: :disappointed: 38 existing translations will be lost with this change.
If the string must change, the following option has already been translated **24** times:
`translate( 'Change your site's address.' )` *ES Score: 9*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

File: client/my-sites/site-settings/delete-site-options/index.jsx [hunk 4]
```diff
@@ -11,68 +11,63 @@ import CompactCard from 'components/card/compact';
...
 
 const trackDeleteSiteOption = ( option ) => {
 	tracks.recordEvent( 'calypso_settings_delete_site_options', {
 		option: option
 	} );
 };
 
-module.exports = React.createClass( {
-	displayName: 'DeleteSite',
+class SiteTools extends React.Component {
```
Comment (tyxla) [inline client/my-sites/site-settings/delete-site-options/index.jsx]: I've seen folks importing `Component` and `PropTypes` from `React` separately to make some statements shorter. How do you feel about doing that here?

File: client/my-sites/site-settings/delete-site-options/index.jsx [hunk 5]
```diff
@@ -11,68 +11,63 @@ import CompactCard from 'components/card/compact';
...
+		const { translate } = this.props;
+
 		const selectedSite = this.props.site;
 		const changeAddressLink = `/domains/manage/${ selectedSite.slug }`;
 		const themeSetupLink = `/settings/theme-setup/${ selectedSite.slug }`;
 		const startOverLink = `/settings/start-over/${ selectedSite.slug }`;
 		const deleteSiteLink = `/settings/delete-site/${ selectedSite.slug }`;
-		let changeAddressLinkText = this.translate( 'Register a new domain or change your site\'s address.' );
-		const themeSetupLinkText = this.translate( 'Make your site look like your theme\'s demo.' );
+		let changeAddressLinkText = translate( 'Register a new domain or change your site\'s address.' );
```
Comment (tyxla) [inline client/my-sites/site-settings/delete-site-options/index.jsx]: If we're changing `site's address` to `site address` below, we should probably change it here too?

File: client/my-sites/site-settings/delete-site-options/index.jsx [hunk 6]
```diff
@@ -102,45 +97,37 @@ module.exports = React.createClass( {
...
-							{ this.translate(
+							{ translate(
 								'All your posts, images, and data will be deleted. ' +
-								'And this site’s address ({{siteAddress /}}) will be lost.',
-								{
-									components: {
-										siteAddress: <strong>{ selectedSite.wpcom_url || selectedSite.slug }</strong>
-									}
-								}
+								'And this site’s address will be lost.'
```
Comment (tyxla) [inline client/my-sites/site-settings/delete-site-options/index.jsx]: If we're changing site's address to site address above, we should probably change it here too?

File: client/my-sites/site-settings/delete-site-options/index.jsx (hunk 3 above)
Comment (tyxla) [inline client/my-sites/site-settings/delete-site-options/index.jsx]: If we decide to change this, we should also make sure to update the existing string.

File: client/my-sites/site-settings/delete-site-options/index.jsx (hunk 2 above)
Comment (tyxla) [inline client/my-sites/site-settings/delete-site-options/index.jsx]: Let's also make sure to update the existing string to keep all existing translations.

File: client/my-sites/site-settings/delete-site-options/index.jsx (hunk 1 above)
Comment (tyxla) [inline client/my-sites/site-settings/delete-site-options/index.jsx]: That looks like a good suggestion, doesn't it?

File: client/my-sites/site-settings/delete-site-options/index.jsx [hunk 7]
```diff
@@ -11,68 +11,63 @@ import CompactCard from 'components/card/compact';
...
 
 const trackDeleteSiteOption = ( option ) => {
 	tracks.recordEvent( 'calypso_settings_delete_site_options', {
 		option: option
 	} );
 };
 
-module.exports = React.createClass( {
-	displayName: 'DeleteSite',
+class SiteTools extends React.Component {
```
Comment (tyxla) [inline client/my-sites/site-settings/delete-site-options/index.jsx]: Nitpicking: our linter does not yell about empty lines in the beginning of classes, but I think it should. Care to remove this one too?

**Comment by tyxla** [comment]
I think it makes complete sense to have a copy review here, cc @michelleweber.

**Comment by michelleweber** [comment]
"Change site address" looks good. For the other three options, I have some notes:

For "Theme Setup," it'd be great to specify the automagical nature of the feature, like:
"Automatically copy the theme demo's settings to get the same look."

"Empty your content" is an odd phrase -- I'd think "Empty your site" or "Delete your content." And for the description underneath suggest:
"Keep your site address and current theme, but remove all posts, pages, and media so you can start fresh."

Two questions about this one:
- Does this remove widgets and theme customizations as well? If so, I'd specify that in this list. ("...remove all posts, pages, media, widgets, and theme customizations..."
- I wanted to double-check -- does this delete your posts/pages or set them back to draft? That would control whether the description says "delete" or "remove"; I'd like to be as specific as possible.

For "delete your site," suggest editing the description -- the two sentences are a little stilted.
"Delete all your posts, pages, media, and data, and give up your site's address."

File: client/my-sites/site-settings/delete-site-options/index.jsx (hunk 1 above)
Comment (seear) [inline client/my-sites/site-settings/delete-site-options/index.jsx]: 👍

File: client/my-sites/site-settings/delete-site-options/index.jsx (hunk 3 above)
Comment (seear) [inline client/my-sites/site-settings/delete-site-options/index.jsx]: 👍

**Comment by rickybanister** [comment]
Thanks @michelleweber!

This is looking great @seear!

**Comment by seear** [comment]
> Two questions about this one:
Does this remove widgets and theme customizations as well? If so, I'd specify that in this list. ("...remove all posts, pages, media, widgets, and theme customizations..."
I wanted to double-check -- does this delete your posts/pages or set them back to draft? That would control whether the description says "delete" or "remove"; I'd like to be as specific as possible.

@michelleweber Seems this links to another page, with a 'contact support' button, and a link to the support doc with steps to remove exactly these items:
screen shot 2017-05-22 at 16 12 08

I'll go with your suggested "Keep your site address and current theme, but remove all posts, pages, and media so you can start fresh.", unless you disagree?

File: client/my-sites/site-settings/delete-site-options/index.jsx (hunk 5 above)
Comment (seear) [inline client/my-sites/site-settings/delete-site-options/index.jsx]: I think using _site's_ is ok in the longer descriptions (see also the description for _Delete your site_) but not so great in the titles.

File: client/my-sites/site-settings/delete-site-options/index.jsx [hunk 8]
```diff
@@ -11,68 +11,62 @@ import CompactCard from 'components/card/compact';
...
+		let changeAddressLinkText = translate( 'Register a new domain or change your site\'s address.' );
+		const themeSetupLinkText = translate( 'Automatically make your site look like your theme\'s demo.' );
 		const strings = {
-			changeSiteAddress: this.translate( 'Change Site Address' ),
-			themeSetup: this.translate( 'Theme Setup' ),
-			startOver: this.translate( 'Start Over' ),
-			deleteSite: this.translate( 'Delete Site' )
+			changeSiteAddress: translate( 'Change your site address' ),
+			themeSetup: translate( 'Theme setup' ),
+			startOver: translate( 'Delete your content' ),
```
Comment (a8ci18n) [inline client/my-sites/site-settings/delete-site-options/index.jsx]: Hi! I've found a possible matching string that has already been translated **72** times:
`translate( 'Delete all content.' )` *ES Score: 7*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

File: client/my-sites/site-settings/delete-site-options/index.jsx [hunk 9]
```diff
@@ -11,68 +11,62 @@ import CompactCard from 'components/card/compact';
...
+
 		const selectedSite = this.props.site;
 		const changeAddressLink = `/domains/manage/${ selectedSite.slug }`;
 		const themeSetupLink = `/settings/theme-setup/${ selectedSite.slug }`;
 		const startOverLink = `/settings/start-over/${ selectedSite.slug }`;
 		const deleteSiteLink = `/settings/delete-site/${ selectedSite.slug }`;
-		let changeAddressLinkText = this.translate( 'Register a new domain or change your site\'s address.' );
-		const themeSetupLinkText = this.translate( 'Make your site look like your theme\'s demo.' );
+		let changeAddressLinkText = translate( 'Register a new domain or change your site\'s address.' );
+		const themeSetupLinkText = translate( 'Automatically make your site look like your theme\'s demo.' );
```
Comment (a8ci18n) [inline client/my-sites/site-settings/delete-site-options/index.jsx]: Hi! I've found a possible matching string that has already been translated **17** times:
`translate( 'Make your site look like your theme's demo.' )` *ES Score: 10*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

Comment (seear) [inline client/my-sites/site-settings/delete-site-options/index.jsx]: 👍

File: client/my-sites/site-settings/delete-site-options/index.jsx (hunk 8 above)
Comment (seear) [inline client/my-sites/site-settings/delete-site-options/index.jsx]: 👍

**Comment by designsimply** [comment]
> For "delete your site," suggest editing the description -- the two sentences are a little stilted.
"Delete all your posts, pages, media, and data, and give up your site's address."

Would it be going too far to change the heading from "delete your site" to "delete your site permanently"? You might be surprised how many support requests actually come in from users who have deleted permanently who actually meant to empty a site. 🙂

**Comment by seear** [comment]
@designsimply done:

screen shot 2017-05-23 at 12 03 03

File: client/my-sites/site-settings/delete-site-options/index.jsx [hunk 10]
```diff
@@ -11,68 +11,62 @@ import CompactCard from 'components/card/compact';
...
 		const strings = {
-			changeSiteAddress: this.translate( 'Change Site Address' ),
-			themeSetup: this.translate( 'Theme Setup' ),
-			startOver: this.translate( 'Start Over' ),
-			deleteSite: this.translate( 'Delete Site' )
+			changeSiteAddress: translate( 'Change your site address' ),
+			themeSetup: translate( 'Theme setup' ),
+			startOver: translate( 'Delete your content' ),
+			deleteSite: translate( 'Delete your site permanently' )
```
Comment (a8ci18n) [inline client/my-sites/site-settings/delete-site-options/index.jsx]: Hi! I've found a possible matching string that has already been translated **80** times:
`translate( 'Delete My Site Permanently' )` *ES Score: 12*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

Comment (seear) [inline client/my-sites/site-settings/delete-site-options/index.jsx]: 👍

**Comment by seear** [comment]
Thanks for the copy review @michelleweber

**Comment by rickybanister** [comment]
@seear I know this is merged, but @designsimply's comment made me think we should add the word 'forever' to the end of the subtext. It would really drive home the permanence!

**Comment by seear** [comment]
> we should add the word 'forever' to the end of the subtext. It would really drive home the permanence!

Yeah, agree. I can add that to another PR, no problem.

**Comment by supernovia** [comment]
_FOREVER_.  With red text on a black background. Just kidding. Sort of.

But in reality I kind of like the word Deactivate vs Delete. Delete implies the name no longer exists, and thus could exist again as though it were brand new, when really it can't.  What do you think?

> Deactivate your site permanently
> Delete all your posts, pages, media, and data, and deactivate your site's address forever.

**Comment by rickybanister** [comment]
I would leave 'delete' in the header, but deactivate your site address could work in the description.

Could you share, at least anecdotally how often someone deletes their site and then wants it back?

**Comment by supernovia** [comment]
Obviously this isn't perfect, but here's a sampling of forum posts with the word "deleted" - and only the ones Google picked up on this week: https://www.google.com/#q=site:en.forums.wordpress.com+deleted&tbs=qdr:w

We get more requests privately.

**Comment by supernovia** [comment]
Also I have serious doubts that some people will read past the header. Just people being people.

**Comment by designsimply** [comment]
<blockquote>Could you share, at least anecdotally how often someone deletes their site and then wants it back?</blockquote>

In my (very) past experience, I'd say maybe once a week at least. This is also a case where you should consider severity in addition to frequency. Restoring a deleted blog has not been allowed in most cases in the past and that makes misunderstandings surrounding mistakenly deleted sites additionally painful to work through for both users and happiness engineers.

To get a feel for it, I would recommend any developer or designer working on a feature like this to do a mini support rotation (i.e. answer just a few related support requests) asking to fix this so that you have a better understanding for what is happening—happiness engineers can help explain it, however you simply won't be able to get a proper empathy for an issue like this without being closer to it.

**Comment by rickybanister** [comment]
Yes, that's always good advice @designsimply 

I guess in my head I'm trying to weigh the following:

We are discussing copy improvements while acknowledging that users often skim or don't read interfaces. I don't particularly feel that 'deactivate' sounds more permanent than 'delete'. Delete feels more permanent to me, so obviously this is a subjective thing. We can add the words 'permanent' and 'forever' to this, but perhaps we should just implement a double confirmation instead.

We already have a confirmation box here, but perhaps we should add an interstitial that says 'Are you sure, you will never ever ever be able to get that site address back and all of your content will be gone forever blah blah blah' with a 'proceed' call to action. Only on the next (third) step could you type in 'delete' or perhaps we make it 'delete forever' to finally confirm. 

That might at least slow things down.

**Comment by supernovia** [comment]
I can say as a team we run into multiple requests per day. Hope that helps. :)

**Pull request merged**